### DISCUSSION TITLE
Need help with a webview.

### DISCUSSION THREAD

**Discussion by no-name-6655** [3kh0/ext-remover#1354] [Webview JOIN TITANIUM NETWORK TO SHARE PLEASE]
I found a new webview but it's not very good, but the reason i'm posting it is because i think it could possibly lead to a pretty good one, because it seems there is may be a way to have an unenrolled browser.
......................................
Step 1: Powerwash

Step 2: You need to trigger an enrollment error. I found you can do this by rapidly switching wifi networks while the chromebook is loading the "Enterprise Enrollment" thing, but it might take a while of spamming.

Step 3: From the error you should be able to hit a button that will let you enroll manually, do that.

-From here it is the same process as a chrome://chrome-signin webview, using microsoft login. If you know that one then do it, then if not just read below

Step 4: In the email put google@d11.org
Step 5: Click 'sign-in options', then click 'sign in with github'
Step 6: Click Privacy at the bottom. From the privacy page, scroll all the way down and click on 'Ask the github community'
Step 7: Now you are at github, from here you just need to find a link to [www.google.com](url) . There's a lot of ways to find one, or you can make a repo with a link on it.

From there you have a webview, but it's not very good as you can't really go forward or back pages, and there's no tabs.

...................................
The part i'm most interested though, is if you go to google on the webview, and sign in with a google account, it will attempt to sign in to the chromebook, not the webview, but it will fail in 1 of 2 ways::

If you sign in with a non-enterprise approved account: 
it will move the screen for a second and then boot you back to the setup. You can prevent it from resetting you  to the start by pressing Alt+Shift+S and opening a wifi options popup. From here I don't know if there is anything else. 

If you use an enterprise approved account:
You will get some error about not being able to retrieve an access token, or something like that. Pressing continue gets you back to the manual enroll

If anyone can test and see if there's anything more from this maybe could lead to a way to get a skiovox-like browser? I'm not sure.

**Comment by S-PScripts** [comment]
is this thing like skiovox even worth it though?
(tbh i'd say the same to literally any unenrollment exploit)

**Reply by no-name-6655** [reply]
you're right its probably not worth, but the point is  for me to see how much you can get around the filters and restrictions, mostly just for fun. If someone just wants to play games on their chromebook at school they could just use a proxy or some unblocked games site, which are plenty effective compared to doing some complicated exploit to get around policies or downgrade their chromebook. But for me and probably some other peoples its just fun to bypass or exploit the things put in place to prevent it.

**Comment by Exploiter-org** [comment]
It works
